services/app-api/handlers/reports/get.ts

```diff
@@ -0,0 +1,38 @@
+import handler from "../handler-lib";
+import dynamoDb from "../../utils/dynamo/dynamodb-lib";
+import { StatusCodes } from "../../utils/types/types";
+import { NO_KEY_ERROR_MESSAGE } from "../../utils/constants/constants";
+
+export const getReport = handler(async (event, _context) => {
+  if (!event?.pathParameters?.reportId!) {
+    throw new Error(NO_KEY_ERROR_MESSAGE);
+  }
+  const statusParams = {
+    TableName: process.env.REPORT_STATUS_TABLE_NAME!,
+    Key: {
+      key: event.pathParameters.reportId,
+    },
+  };
+  const reportParams = {
+    TableName: process.env.REPORT_TABLE_NAME!,
+    Key: {
+      key: event.pathParameters.reportId,
+    },
+  };
+  const statusQueryResponse = await dynamoDb.get(statusParams);
+  const reportQueryResponse = await dynamoDb.get(reportParams);
+
+  const statusItem =
+    typeof statusQueryResponse.Item === "object"
```

Comment:
For context here is the error I was trying to avoid: 
<img width="468" alt="Screen Shot 2022-07-27 at 7 21 36 AM" src="https://user-images.githubusercontent.com/57802560/181271404-b45c78ac-5f74-4918-b81d-467e27798b5b.png">

Reply:
The other option is checking if it exists, then casting it. This works, I didn't realize it would cause a type error.